src/organization/organization.action.js

```diff
@@ -1,68 +1,80 @@
-import { fetchOrg, fetchOrgMembers, fetchUrl } from 'api';
-import { GET_ORG, GET_ORG_REPOS, GET_ORG_MEMBERS } from './organization.type';
+import { createAction } from 'redux-actions';
 
-export const getOrg = orgName => {
-  return (dispatch, getState) => {
-    const accessToken = getState().auth.accessToken;
+import {
+  fetchOrg,
+  fetchOrgMembers,
+  fetchUrl,
+} from 'api';
+import {
+  GET_ORG,
+  GET_ORG_LOADING,
+  GET_ORG_ERROR,
+  GET_ORG_REPOS,
+  GET_ORG_REPOS_LOADING,
+  GET_ORG_REPOS_ERROR,
+  GET_ORG_MEMBERS,
+  GET_ORG_MEMBERS_LOADING,
+  GET_ORG_MEMBERS_ERROR,
+} from './organization.constants';
 
-    dispatch({ type: GET_ORG.PENDING });
+export const getOrg = createAction(GET_ORG);
```

Comment:
Creating actions by using [redux-actions ](https://github.com/reduxactions/redux-actions) I think this is more organized than have a manual dispatch with some boilerplate like `{type: ... , payload: ...}`